pkg/geodata/rowquery_test.go

```diff
@@ -0,0 +1,80 @@
+// +build comptest
+
+package geodata
+
+import (
+	"context"
+	"flag"
+	"log"
+	"testing"
+
+	"github.com/ONSdigital/dp-find-insights-poc-api/comptests"
+	"github.com/ONSdigital/dp-find-insights-poc-api/model"
+	"github.com/ONSdigital/dp-find-insights-poc-api/pkg/database"
+)
+
+const dsn = "postgres://insights:insights@localhost:54323/censustest"
+
+// passing -args -kill=false to the test will kill docker postgres
+var kill = flag.Bool("kill", false, "docker kill postgres")
+
+// TODO Check empty
+func TestRowQuery(t *testing.T) {
+	comptests.SetupDockerDB(dsn)
+	model.SetupDB(dsn)
+
+	db, err := database.Open("pgx", dsn)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	func() {
+		db.DB().Exec("BEGIN")
+		defer db.DB().Exec("ROLLBACK")
+
+		// setup database
+		doSQL(t, db, "INSERT INTO GEO_TYPE (id,name) VALUES (6,'LSOA')")
+		doSQL(t, db, "INSERT INTO GEO (id,type_id,code,name,lat,long,valid,wkb_geometry,wkb_long_lat_geom) VALUES (7562,6,'E01000001','City of London 001A',51.5181,-0.09706,true,'0103000020E6100000010000000600000099B7D26A2C41B8BFCABAA1E4A2C249403E6F5814C06FB8BF4B7A3CFEF9C149407A98D8707087B9BF11B32F7F25C249403FBA9D552F37B9BF7CB1250DA1C24940ECB58D3C65E6B8BF8C2B844AC3C2494099B7D26A2C41B8BFCABAA1E4A2C24940','0101000020E610000045F0BF95ECD8B8BF5F07CE1951C24940')")
```

Comment:
v. long string here - not sure if thats a fixable issue or not

Reply:
gofmt thought it ok but I'll tweak it.

Reply:
we don't need the geography types in this test set to null